app/src/main/java/com/kickstarter/ui/activities/compose/search/AmountRaisedSheet.kt

```diff
@@ -0,0 +1,198 @@
+
+    KSTheme {
+        Surface(
+            color = colors.backgroundSurfacePrimary
+        ) {
+            Column(
+                modifier = Modifier
+                    .fillMaxWidth()
+            ) {
+                Row(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .drawBehind {
+                            drawLine(
+                                color = backgroundDisabledColor,
+                                start = Offset(0f, size.height),
+                                end = Offset(size.width, size.height),
+                                strokeWidth = dimensions.dividerThickness.toPx()
+                            )
+                        }
+                        .padding(top = dimensions.paddingLarge, bottom = dimensions.paddingLarge, end = dimensions.paddingMediumSmall),
+
+                    verticalAlignment = Alignment.CenterVertically
+                ) {
+                    KSIconButton(
+                        modifier = Modifier
+                            .padding(start = dimensions.paddingSmall)
+                            .testTag(SearchScreenTestTag.BACK_BUTTON.name),
+                        onClick = onNavigate,
+                        imageVector = Icons.AutoMirrored.Filled.ArrowBack,
+                        contentDescription = stringResource(id = R.string.Back)
+                    )
+
+                    Text(
+                        text = stringResource(R.string.Amount_raised_fpo),
+                        style = typographyV2.headingXL,
+                        modifier = Modifier.weight(1f),
+                        color = colors.textPrimary
+                    )
+
+                    KSIconButton(
+                        modifier = Modifier.testTag(CategorySelectionSheetTestTag.DISMISS_BUTTON.name),
+                        onClick = onDismiss,
+                        imageVector = Icons.Filled.Close,
+                        contentDescription = stringResource(id = R.string.accessibility_discovery_buttons_close)
+                    )
+                }
+
+                LazyColumn(
+                    modifier = Modifier
+                        .testTag(BUCKETS_LIST)
+                        .fillMaxWidth()
+                        .weight(1f)
+                        .drawBehind {
+                            drawLine(
+                                color = backgroundDisabledColor,
+                                start = Offset(0f, size.height),
+                                end = Offset(size.width, size.height),
+                                strokeWidth = dimensions.dividerThickness.toPx()
+                            )
+                        }
+                        .padding(horizontal = dimensions.paddingLarge, vertical = dimensions.paddingMedium),
+                ) {
+                    val validBuckets = DiscoveryParams.AmountBuckets.values()
```

Comment:
ℹ️  Potential refactor to take a loot at once we have less deadlines, generics for this specific screen, so we can make it reusable for phases 4, 5 and 6,  (RaisedBuckets, PledgedBuckets, GoalBuckets)